src/util.js

```diff
@@ -1985,3 +1962,65 @@ export const hashThought = key =>
 
 export const getThought = (key, data = store.getState().data) =>
   data[hashThought(key)]
+
+export const initState = async () => {
```

Comment:
Having both `initState` and `initialState` is very confusing. Let's come up with a more coherent strategy where we either combine them or come up with better names.